sqlx-sqlite/src/connection/mod.rs

```diff
@@ -544,3 +549,219 @@ impl Statements {
         self.temp = None;
     }
 }
+
+#[cfg(feature = "preupdate-hook")]
+mod preupdate_hook {
```

Comment:
Done. I considered moving all the hooks into their own module since there's quite a few now, but that would be a larger change. I'm happy to do that if you'd like though.